### PR TITLE
Add Input class per issue #8

### PR DESCRIPTION
**Pull request by ivantot**

### CODE REVIEW

File: HECalculator/Backend/Worker.cs [hunk 1]
```diff
@@ -5,4 +5,6 @@ public class Worker
 	public Worker()
 	{
 	}
+
+	private Inputs inputs;
```
Comment (VuceticBranislav) [inline HECalculator/Backend/Worker.cs]: Add generic getter and setter

**Comment by VuceticBranislav** [comment]
Also remove Dummy class from  [Form1.cs](https://github.com/HECalculator/HECalculator-Desk/blob/3ef770a1d80a348bcf11088a2c3e2c119db1c000/HECalculator/Form1.cs#L29-L38)

File: HECalculator/Form1.cs [hunk 2]
```diff
@@ -20,9 +20,9 @@ public Form1()
...
         {
             // Process input and generate output exsample using custom class
-            var dummy = new Dummy();
-            dummy.Value = textBox1.Text;
-            textBox2.Text = dummy.Value;
+            var dummy = new Inputs();
+            dummy.MatDesc = textBox1.Text;
+            textBox2.Text = dummy.MatDesc;
         }
     }
```
Comment (VuceticBranislav) [inline HECalculator/Form1.cs]: Remove Dummy class from file.

**Review by VuceticBranislav — CHANGES_REQUESTED** [review]
Update review comments to close PR

File: HECalculator/Backend/Worker.cs (hunk 1 above)
Comment (ivantot) [reply HECalculator/Backend/Worker.cs]: Should discuss a standard of making getters and setters (auto property, traditional...)

Comment (VuceticBranislav) [reply HECalculator/Backend/Worker.cs]: Discussion can be found here #13

**Pull request merged**